package.json

```diff
@@ -387,6 +387,12 @@
     "embedding-sdk:generate-changelog": "yarn conventional-changelog -p angular --pkg enterprise/frontend/src/embedding-sdk/package.template.json --config enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js --tagPrefix embedding-sdk-",
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
     "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
+    "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json",
+    "embedding-sdk:clean": "rm -rf resources/embedding-sdk/dist && rm -rf resources/embedding-sdk/.tsbuildinfo",
+    "embedding-sdk:dev": "concurrently -n 'tsc,webpack,fix-types' -c 'blue,green,yellow' 'npm:embedding-sdk:dev:dts' 'npm:embedding-sdk:dev:webpack' 'npm:embedding-sdk:dev:fixup'",
+    "embedding-sdk:dev:dts": "yarn embedding-sdk:tsc --watch --preserveWatchOutput",
+    "embedding-sdk:dev:webpack": "SKIP_DTS=true yarn build-embedding-sdk:watch",
+    "embedding-sdk:dev:fixup": "yarn embedding-sdk:fixup-types-imports --watch",
```

Comment:
I think you made a similar comment before. This PR adds 6 new package.json scripts, and it's overwhelming for me to know which one I need to use. I couldn't think I would need all 6 just to run the SDK in development, not including the fact that we also still have the old commands lying around.

I'd love to see some consolidation, deletion, and make our command easy to understand and easy to use more.

Reply:
I 100% agree with you here, the reason why they're separate scripts is so that they can run on different terminals (ideally, with vscode you just run the task and it runs them without you having to know which ones to run).
I can add some quick doc about these commands and then we could hopefully migrate to our own package.json, but that would be its own task

Reply:
I think we shouldn't optimize only for VS code users. Having the ability to run on VS Code is great, but not everybody is running VS Code. I'd like to see these commands being optimized for a command line, then maybe as a by-product being able to run in VS Code.

But for now, maybe a quick doc would do. Let's see.